packages/noya-state/src/reducers/themeReducer.ts

```diff
@@ -307,32 +311,19 @@ export function themeReducer(
         draft.sketch.document.layerTextStyles = textStyles;
       });
     }
-    case 'selectSwatch': {
-      const [, id, selectionType = 'replace'] = action;
-
-      return produce(state, (draft) => {
-        updateSelection(draft.selectedSwatchIds, id, selectionType);
-      });
-    }
-    case 'selectThemeStyle': {
-      const [, id, selectionType = 'replace'] = action;
-
-      return produce(state, (draft) => {
-        updateSelection(draft.selectedLayerStyleIds, id, selectionType);
-      });
-    }
-    case 'selectTextStyle': {
-      const [, id, selectionType = 'replace'] = action;
-
-      return produce(state, (draft) => {
-        updateSelection(draft.selectedTextStyleIds, id, selectionType);
-      });
-    }
+    case 'selectSwatch':
+    case 'selectThemeStyle':
+    case 'selectTextStyle':
     case 'selectSymbol': {
       const [, id, selectionType = 'replace'] = action;
 
+      const crrThemeTab = state.currentThemeTab;
```

Comment:
Rename `crrThemeTab` => `currentThemeTab`

We don't abbreviate names much, and good to keep the names consistent with what we already have